lib/inspector.coffee

```diff
@@ -50,7 +56,7 @@ class Inspector
             cursor_pos = cursor.getBufferColumn()
         return [code, cursor_pos]
 
-    getInspector: ->
+    initializeInspector: ->
```

Comment:
I'm OK with renaming `@inspector` to `@view`. But I don't think `@initializeInspector` should be renamed to `@show` since it just constructs or clears the view. The inspector is only shown if a message is added.

I didn't move the creation to the constructor because the `inspect` command is only used on occasions, hence I think the view should only be created on the first `inspect` request. Or am I missing something?